srcpkgs/swaync/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'swaync'
+pkgname=swaync
+version=0.7.2
+revision=1
+wrksrc=SwayNotificationCenter-${version}
+build_style=meson
+hostmakedepends="vala glib-devel pkg-config"
+makedepends="gobject-introspection gtk+3-devel gtk-layer-shell-devel
+json-glib-devel libhandy1-devel scdoc"
```

Comment:
scdoc shouuld definitely be hostmakedepend

Reply:
updated.